active/progress/ProgressRing2.md

```diff
@@ -0,0 +1,146 @@
+* the [ProgressBar](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ProgressBar)
+control, which shows progress on a rectangle
+* and the
+[ProgressRing](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ProgressRing)
+control, which shows progress on a circle.
+
+Progress _Bar_ has both a determinate and indeterminate mode.
+In determine mode the rectangle goes from empty to full, in indeterminate mode it shows an animation.
+Progress _Ring_ currently only has an indeterminate mode, but this spec is adding an indeterminate mode.
+
+ProgressBar is also getting a feature to replaces its built-in visualization (a spinning ring)
+with a custom animation defined as an 
+[IAnimatedVisualSource](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.IAnimatedVisualSource),
+which is known as a "Lottie Animation", and already supported (playable by) an
+[AnimatedVisualPlayer](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer).
+
+# ProgressRing Description
+Represents a control that indicates that an operation is ongoing. 
+The typical visual appearance is a ring-shaped "spinner" that animates a filled area as progress continues.
+
+By default, ProgressRing will display an indeterminate spinner that continuously 
+animates until the control is no longer visible ?? or 
+[ProgressRing.IsActive](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.ProgressRing.IsActive)
+is set to false. You can change the behavior to display
+a determinate ring that fills in based on the value you provide, using the `IsIndeterminate` property.
+
+You can also replace the built-in animation shown by the ProgressRing with a custom one,
+using the `DeterminateSource` and `IndeterminateSource` properties.
+
+# Examples
+
+The following examples show how to use the `IsIndeterminate` property to change the mode of 
+the ProgressRing and the Value property to change the proportionate amount indicated. 
+
+## Indeterminate ProgressRing
+
+By default ProgressRing is indeterminate (a spinning circle).
+
+> Issue: not the same without an AniGif
+
+```xml
+<ProgressRing IsActive="True" />
+```
+![](images/ProgressRing-indeterminate.jpg)
+
+## Determinate ProgressRing
+
+When `IsIndeterminate` is set, the circle will show progress around the ring,
+filling it when Value gets to Maximum.
+
+```xml
+<ProgressRing IsActive="True" IsIndeterminate="False"
+    Minimum="32" Maximum="212" Value="{x:Bind LiquidTemp}" />
+```
+![](images/ProgressRing-determinate.PNG)
+
+
+## Custom animation
+
+Show progress ring that visually looks like a thermometer boiling over.
+See [AnimatedVisualPlayer](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer)
+for more information on animated visuals.
+
+> Picture would be nice
+
+```xml
+<ProgressRing IsActive="True" >
+    <ProgressRing.IndeterminateSource>
+        <animatedvisuals:LottieProgress />
+    </ProgressRing.IndeterminateSource>
```

Comment:
If an indeterminate ProgressRing becomes inactive, what happens to the animation? Is it paused at its current position? Does it rewind to 0? Does it become invisible (but occupy space)?